core/templates/pages/exploration-editor-page/improvements-tab/templates/feedback-thread-modal.template.html

```diff
@@ -1,7 +1,7 @@
-<div class="modal-header" style="display: block">
-  <h3 style="float: left"><[getTitle()]></h3>
+<div class="modal-header oppia-feedback-thread-header">
+  <h3 class="oppia-get-title"><[getTitle()]></h3>
```

Comment:
Use float-left.

Please change here and throughout your changes. Ditto for float-right below.